src/olympia/translations/tests/test_models.py

```diff
@@ -481,7 +486,7 @@ def test_translations_queries(self):
         # Make sure we are in a clean environnement.
         self.reset_queries()
         TranslatedModel.objects.get(pk=1)
-        assert len(connections['default'].queries) == 3
+        assert len(connections['default'].queries) == 2
```

Comment:
Locally, this fails for me, doubly so: I get 4 queries. Other tests fail with 5 or 4 queries as well !

The reason why there should be 3 queries is because of the way cache-machine caches things:
- 1 to fetch the ids of the `TranslatedModel` objects (`FETCH_BY_ID` mode, which should be enabled)
- 1 to fetch the `TranslatedModel` themselves, without the translations
- 1 to fetch the translations

That should still be the case regardless of the number of objects fetched, btw.

Reply:
Detail of that failure with 4 queries:
Failing test is `TranslationMultiDbTests.test_translations_queries`
`connections['default'].queries[0]` and `connections['default'].queries[1]` are the ones I describe above, fetching the ids then the actual `TranslatedModel` objects.
`connections['default'].queries[2]` and `connections['default'].queries[3]` are... identical. The query itself looks ok, matching what we expect for the query that fetches the translations, it's just that it's repeated twice - maybe because there are 2 fields on the model. That's not good.

Reply:
This is super weird, I'll look into why those are not failing in the same way for me.

Reply:
Alrighty, short update: I now see these failures locally too, I'll debug this more